### PR TITLE
amam/round_up_barrier

### PR DESCRIPTION
**Pull request by cakasuma**

### CODE REVIEW

File: src/trade/tradeConf.es6 [hunk 1]
```diff
@@ -53,7 +53,7 @@ rv.binders['tick-chart'] = {
                   align: 'left',
                   x: 0,
                   formatter() {
-                        return addComma(this.value.toFixed(display_decimals));
+                        return this.value;
```
[the remark this answers is not in the document]
Comment (cakasuma) [reply src/trade/tradeConf.es6]: the addcomma function is to make the value price formatted. but true, i have to use `toFixed(display_decimals) to make the fractional length of value correct

[the remark this answers is not in the document]
Comment (OskarAhl) [reply src/trade/tradeConf.es6]: I think we want to keep the `addComma` to have the numbers comma formatted e.g. `123,456.34` else the numbers will be `123456.34`?

[the remark this answers is not in the document]
Comment (cakasuma) [reply src/trade/tradeConf.es6]: in binary static it shows `123456.34` ?

[the remark this answers is not in the document]
Comment (OskarAhl) [reply src/trade/tradeConf.es6]: Ah the barrier yes, but not in the tooltip

[the remark this answers is not in the document]
Comment (morteza-binary) [reply src/trade/tradeConf.es6]: I agree with @OskarAhl we should keep the `addComma`.

File: src/trade/tradeConf.es6 [hunk 2]
```diff
@@ -38,7 +38,8 @@ rv.binders['tick-chart'] = {
             formatter: function () {
                const tick = model.array[this.x - 1];
                if (tick && tick.tooltip) {
-                     return `<div class='tooltip-body'>${tick.tooltip}</div>`;
+                  console.log(tick.tooltip);
```
Comment (OskarAhl) [inline src/trade/tradeConf.es6]: Can we remove this console log too?

Comment (cakasuma) [reply src/trade/tradeConf.es6]: sure

**Pull request merged**